odoo/addons/base/models/res_lang.py

```diff
@@ -332,6 +332,21 @@ def format(self, percent, value, grouping=False, monetary=False):
 
         return formatted
 
+    def copy(self, default=None):
```

Comment:
Oops, sorry I showed you the wrong cheat code :see_no_evil: it's better to override `copy_data` if possible. Note that it was changed in saas-17.2 to handle the multi-record case.